### PR TITLE
Create ihatemoney.yml

### PR DESCRIPTION
**Pull request by almet**

### CODE REVIEW

File: _data/projects/ihatemoney.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+desc: A simple shared budget manager web application
+site: https://github.com/spiral-project/ihatemoney
+tags:
+- python
+- flask
+- budget
+- rest
+upforgrabs:
+  name: good first issue
+  link: https://github.com/spiral-project/ihatemoney/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22
```
Comment (ritwik12) [inline _data/projects/ihatemoney.yml]: @almet Please change this to https://github.com/spiral-project/ihatemoney/labels/good%20first%20issue

**Review by ritwik12 — CHANGES_REQUESTED** [review]

File: _data/projects/ihatemoney.yml (hunk 1 above)
Comment (almet) [reply _data/projects/ihatemoney.yml]: Here we go!

**Review by ritwik12 — APPROVED** [review]

**Comment by ritwik12** [comment]
@shiftkey Did you made gh-pages a protected branch? I am not able to merge anymore. :(

**Comment by ritwik12** [comment]
Screenshot 2019-10-14 at 11 58 46 AM

**Comment by shiftkey** [comment]
@ritwik12 should be good now - I just missed a permission thing

**Pull request merged**